### PR TITLE
ETC/QUEST/QUESTLV100: RTT review of early game lines

### PR DESCRIPTION
**Pull request by ttgmichael**
RTT of the early game lines.

Let me know if I need to fix something.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -1216,20 +1216,20 @@ QUEST_LV_0100_20150317_001215	I firmly believe that the Goddess will return some
...
-QUEST_LV_0100_20150317_001228	Have you ever seen Vubbes using pickaxes? {nl} What are they trying to find...
-QUEST_LV_0100_20150317_001229	Thank you very much for saved us.{nl}
-QUEST_LV_0100_20150317_001230	If you still want to enter the Closed Area, collect the strange stone from the Vubbes. {nl} Stic it to the wall and the crystal wals open.
-QUEST_LV_0100_20150317_001231	What are you doing here? {nl} Go into the mines quickly. {nl} Weren't we supposed to meet in 1st Gallery?
-QUEST_LV_0100_20150317_001232	No demon would be able to cross here.{nl}I will do my best to protect this place so they can't interrupt the Paladin Master.
+QUEST_LV_0100_20150317_001228	$Have you ever seen Vubbes using pickaxes? {nl} I wonder what they are trying to find...
+QUEST_LV_0100_20150317_001229	$Thank you very much for saved us.{nl}
+QUEST_LV_0100_20150317_001230	$If you still want to enter the Closed Area, collect Vubbe Magic Stones from the Vubbes. {nl} And then place them inside a gap and the Crystal Wall should open.
+QUEST_LV_0100_20150317_001231	$What are you doing here? {nl} Hurry and get into the mines. {nl} Weren't we supposed to meet in the 1st Floor?
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Uncapitalize the F in floor? Since it's a normal sentence.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -5634,33 +5634,33 @@ QUEST_LV_0100_20150317_005633	Look for the Goddess in Mage Tower 5th Floor
...
-QUEST_LV_0100_20150317_005651	Found the level handle clamp in the grass. Go back to Gilbert. 
-QUEST_LV_0100_20150317_005652	Talk to the Guard
-QUEST_LV_0100_20150317_005653	Give sugar beets to Baby Pantos
+QUEST_LV_0100_20150317_005639	$Defeat Grummers and Zignuts
+QUEST_LV_0100_20150317_005640	$Gilbert wants you to defeat the monsters that destroyed the cable car. Defeat them in Cholras Winding Road and report back Gilbert about it. 
+QUEST_LV_0100_20150317_005641	Punished the Grummers and Zignuts. Tell Gilbert you taught them a lesson. 
+QUEST_LV_0100_20150317_005642	$Defeat Zignuts
+QUEST_LV_0100_20150317_005643	$Defeat Grummers
+QUEST_LV_0100_20150317_005644	$Defeat the Pantos and retrieve the Gear
+QUEST_LV_0100_20150317_005645	$Gilbert said that Pantos in Mieguista Slope stole cable car's Gear. Defeat them to take back the Gear and return it to Gilbert.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Uncapitalize the G in gear since it's a normal sentence? :x

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -6851,8 +6851,8 @@ QUEST_LV_0100_20150428_006850	The assistant commander Volda asked you to defeat
 QUEST_LV_0100_20150428_006851	Find the person who knows about the epitaph of Gailla Flurry and listen to his request.
 QUEST_LV_0100_20150428_006852	You found the luggage of the villager. Return the luggage to the villager and ask him about the contents in the epitaph.
 QUEST_LV_0100_20150428_006853	Defeat Blue Woodspirits that appeared near Obelisk
-QUEST_LV_0100_20150428_006854	$Goddess Saule lost the revelation from Bramble at the thorny forest of Kvailas Forest. Help the priests at Gate Route and Sirdgela Forest and look for the revelation at Kvailas Forest.
-QUEST_LV_0100_20150428_006855	Take the cable car from Srautas Canyon that will appear as you go down from the twin bridge of the crystal mine, and go up a bit to get to Gelly Plauteau.
+QUEST_LV_0100_20150428_006854	$The Goddess Saule lost the revelation from Brambles at the thorny forest of Kvailas forest. Help the priests at Gate Route and Sirdgela Forest and look for the revelation at Kvailas Forest.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Brambles -> Bramble - Bramble is the boss monster.
Kvailas Forest is a map name though, so why the lowercase f?

also

thorny forest -> Thorn Forest

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -1216,20 +1216,20 @@ QUEST_LV_0100_20150317_001215	I firmly believe that the Goddess will return some
...
-QUEST_LV_0100_20150317_001230	If you still want to enter the Closed Area, collect the strange stone from the Vubbes. {nl} Stic it to the wall and the crystal wals open.
-QUEST_LV_0100_20150317_001231	What are you doing here? {nl} Go into the mines quickly. {nl} Weren't we supposed to meet in 1st Gallery?
-QUEST_LV_0100_20150317_001232	No demon would be able to cross here.{nl}I will do my best to protect this place so they can't interrupt the Paladin Master.
+QUEST_LV_0100_20150317_001228	$Have you ever seen Vubbes using pickaxes? {nl} I wonder what they are trying to find...
+QUEST_LV_0100_20150317_001229	$Thank you very much for saved us.{nl}
+QUEST_LV_0100_20150317_001230	$If you still want to enter the Closed Area, collect Vubbe Magic Stones from the Vubbes. {nl} And then place them inside a gap and the Crystal Wall should open.
+QUEST_LV_0100_20150317_001231	$What are you doing here? {nl} Hurry and get into the mines. {nl} Weren't we supposed to meet in the 1st floor?
+QUEST_LV_0100_20150317_001232	$No demons should be able to pass here.{nl}I will do my best to protect here so they can't interrupt the Paladin Master.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: No demon should be able to pass here.

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -297,7 +297,7 @@ QUEST_LV_0100_20150317_000296	$Stamina is consumed when you move.{nl}You can rec
 QUEST_LV_0100_20150317_000297	$I heard news from my men and was becoming worried.{nl}All is well if it ended up good.
 QUEST_LV_0100_20150317_000298	$Klaipeda Resident
 QUEST_LV_0100_20150317_000299	$)} {nl}When do you think the Goddess is coming back?{nl}Even if I ask the Revelators, all they say is that they must go to Klaipeda...
-QUEST_LV_0100_20150317_000300	$Are you a Revelator?{nl}Didn't the Goddess appeared in your dream?{nl}I've heard so from the soldiers.
+QUEST_LV_0100_20150317_000300	Are you a Revelator?{nl}Didn't a Goddess appear to you in a dream?{nl}I heard so from the soldiers.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: Sounds odd. Why not just say "Did a Goddess appear in your dream"?

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -364,8 +364,8 @@ QUEST_LV_0100_20150317_000363	It's a single road from here on so you'll be able
 QUEST_LV_0100_20150317_000364	Oh, let me give you some helpful information in return for your your help.{nl}The evidence of salvation that the Revelators are looking for can be found in the Closed Area.
 QUEST_LV_0100_20150317_000365	One thing, there's something that keeps bothering me.{nl}A vague feeling of something alien.
 QUEST_LV_0100_20150317_000366	$Miner
-QUEST_LV_0100_20150317_000367	$Thank you for saving me.{nl}It's like a dream for a revelator to come and save us.{nl}
-QUEST_LV_0100_20150317_000368	$The Closed Area?{nl}If you absolutely need to get in there, gather the mysterious stones that the Vubbes carry.{nl}The crystal wall should open when you touch enough stones to it.
+QUEST_LV_0100_20150317_000367	$Thank you for saving us.{nl}It's like a dream for a revelator has come to save us.{nl}
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: It's like a dream to be saved by a/the Revelator.

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -364,8 +364,8 @@ QUEST_LV_0100_20150317_000363	It's a single road from here on so you'll be able
...
+QUEST_LV_0100_20150317_000368	$The Closed Area?{nl}If you absolutely need to get in there, gather the mysterious stones that the Vubbes carry.{nl}The Crystal Wall should open when you touch enough stones to it.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: 'should open when you touch enough stones to it' sounds odd

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -1153,8 +1153,8 @@ QUEST_LV_0100_20150317_001152	The demons could have used their hands to make it
 QUEST_LV_0100_20150317_001153	Since the divine heal was hidden deep below the ground. {nl} Who could have known that the reason for the world's overturn was because of the plants?
 QUEST_LV_0100_20150317_001154	You may use a bit of force if words aren't enough. {nl}We can't grow any further from each other anymore.
 QUEST_LV_0100_20150317_001155	You can summon the shaman doll with this summon scroll. {nl}
-QUEST_LV_0100_20150317_001156	{memo X}If possible, defeat the Carnivore too. {nl} It was contaminated by evil energy and I can't stand watching it anymore.
-QUEST_LV_0100_20150317_001157	Make a mess in the nest of Poatas in Margas Hill and it will appear. {nl}
+QUEST_LV_0100_20150317_001156	{memo X}If possible, defeat the Carnivore too. {nl} It was contaminated by the Magi and I can't stand watching it anymore.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: It was corrupted by the demons

File: QUEST_LV_0100.tsv (hunk 7 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: I'll use insert then

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -1216,20 +1216,20 @@ QUEST_LV_0100_20150317_001215	I firmly believe that the Goddess will return some
...
 QUEST_LV_0100_20150317_001227	Everything has a source called Mana. {nl} And everyone can connect to each other's Mana if they want. {nl} Just like digging a canal.
-QUEST_LV_0100_20150317_001228	Have you ever seen Vubbes using pickaxes? {nl} What are they trying to find...
-QUEST_LV_0100_20150317_001229	Thank you very much for saved us.{nl}
-QUEST_LV_0100_20150317_001230	If you still want to enter the Closed Area, collect the strange stone from the Vubbes. {nl} Stic it to the wall and the crystal wals open.
-QUEST_LV_0100_20150317_001231	What are you doing here? {nl} Go into the mines quickly. {nl} Weren't we supposed to meet in 1st Gallery?
-QUEST_LV_0100_20150317_001232	No demon would be able to cross here.{nl}I will do my best to protect this place so they can't interrupt the Paladin Master.
+QUEST_LV_0100_20150317_001228	$Have you ever seen Vubbes using pickaxes? {nl} I wonder what they are trying to find...
+QUEST_LV_0100_20150317_001229	$Thank you very much for saved us.{nl}
+QUEST_LV_0100_20150317_001230	$If you still want to enter the Closed Area, collect Vubbe Magic Stones from the Vubbes. {nl} And then place them inside a gap and the Crystal Wall should open.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: It said touch them just prior, now it says place them

File: QUEST_LV_0100.tsv (hunk 5 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: She's asking for confirmation.

So another way of saying this is : A goddess appeared in your dreams, right. 

I'll change the a to your, not sure why I changed it to a o-o.

File: QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -2335,13 +2335,13 @@ QUEST_LV_0100_20150317_002334	Sincerity is our strength. {nl} There will be no p
 QUEST_LV_0100_20150317_002335	I think being confined in there damaged my throat and now my throat hurts. {nl} But what about the girl who was with us? Where did she go?
 QUEST_LV_0100_20150317_002336	Aren't you the Revelator who saved us last time? {nl} We were more than glad to see you back then.
 QUEST_LV_0100_20150317_002337	The Goddess must have sent you.{nl}If that's not it.. we already.. It's too scary to think about it.
-QUEST_LV_0100_20150317_002338	It was horrible in captivity. {nl} The Vubbes would come and smell us.. {nl} and make us dig here and there using a pickaxe..
+QUEST_LV_0100_20150317_002338	It was horrible in captivity. {nl} The Vubbes would come and smelled us.. {nl} and made us dig here and there using pickaxes..
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: It was horrible in captivity. {nl} The Vubbes would come and smell us.. {nl} and make us dig here and there using pickaxes..

File: QUEST_LV_0100.tsv [hunk 11]
```diff
@@ -5634,33 +5634,33 @@ QUEST_LV_0100_20150317_005633	Look for the Goddess in Mage Tower 5th Floor
...
-QUEST_LV_0100_20150317_005650	Pantos in Mieguista Hill stole cable car parts and hid it in the grass. Find the lever handle from the grass. 
-QUEST_LV_0100_20150317_005651	Found the level handle clamp in the grass. Go back to Gilbert. 
-QUEST_LV_0100_20150317_005652	Talk to the Guard
-QUEST_LV_0100_20150317_005653	Give sugar beets to Baby Pantos
+QUEST_LV_0100_20150317_005639	$Defeat Grummers and Zignuts
+QUEST_LV_0100_20150317_005640	$Gilbert wants you to defeat the monsters that destroyed the cable car. Defeat them in Cholras Winding Road and report back Gilbert about it. 
+QUEST_LV_0100_20150317_005641	Punished the Grummers and Zignuts. Tell Gilbert you taught them a lesson. 
+QUEST_LV_0100_20150317_005642	$Defeat Zignuts
+QUEST_LV_0100_20150317_005643	$Defeat Grummers
+QUEST_LV_0100_20150317_005644	$Defeat the Pantos and retrieve the Gear
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: Defeat Pantos to retrieve Gears

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (vtange) [inline QUEST_LV_0100.tsv]: Gilbert said that Pantos in Mieguista Slope stole the cable car's Gear.

File: QUEST_LV_0100.tsv [hunk 12]
```diff
@@ -5634,33 +5634,33 @@ QUEST_LV_0100_20150317_005633	Look for the Goddess in Mage Tower 5th Floor
...
+QUEST_LV_0100_20150317_005645	$Gilbert said that Pantos in Mieguista Slope stole cable car's Gear. Defeat them to take back the Gear and return it to Gilbert. 
+QUEST_LV_0100_20150317_005646	$Got the Gear by defeating the Pantos. Return it to Gilbert. 
+QUEST_LV_0100_20150317_005647	$Defeat Pantos and get %s
+QUEST_LV_0100_20150317_005648	$Defeat Pantos
+QUEST_LV_0100_20150317_005649	$Find the Lever Handle Latch
+QUEST_LV_0100_20150317_005650	$Pantos in Mieguista Slope stole cable car parts and hid them in the grass. Find the Lever Handle Latch from the grass. 
+QUEST_LV_0100_20150317_005651	$Found the Lever Handle Latch in the grass. Go back to Gilbert. 
+QUEST_LV_0100_20150317_005652	Talk to the guard
+QUEST_LV_0100_20150317_005653	Give sugar beets to baby panto
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: Baby Panto

File: QUEST_LV_0100.tsv [hunk 13]
```diff
@@ -5634,33 +5634,33 @@ QUEST_LV_0100_20150317_005633	Look for the Goddess in Mage Tower 5th Floor
...
-QUEST_LV_0100_20150317_005655	Lured the Baby Pantos with sugar beets. Listen to Molly's next plans. 
-QUEST_LV_0100_20150317_005656	Talk to Guard Matthew
-QUEST_LV_0100_20150317_005657	Retrieve cable car parts
-QUEST_LV_0100_20150317_005658	Guard Matthew asked for your help to find cable car parts that the Zignuts swallowed around the Nepavy Area.
-QUEST_LV_0100_20150317_005659	Found all the cable car parts as Matthew asked. Give it to Matthew. 
-QUEST_LV_0100_20150317_005660	Retrieve parts swallowed by Zignuts %s 
-QUEST_LV_0100_20150317_005661	Defeat Poatas that threaten the cable car
-QUEST_LV_0100_20150317_005662	Matthew asked you to defeat the Pantos that might destroy the cable car again. Soil Poata's habitat in Margas Hills and lure them to defeat them. 
-QUEST_LV_0100_20150317_005663	Defeated Poatas as Matthew requested. Tell the good news to Matthew. 
+QUEST_LV_0100_20150317_005655	Lured the baby pantos with sugar beets. Listen to Molly's next plans.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: Baby Panto

File: QUEST_LV_0100.tsv [hunk 14]
```diff
@@ -5634,33 +5634,33 @@ QUEST_LV_0100_20150317_005633	Look for the Goddess in Mage Tower 5th Floor
...
-QUEST_LV_0100_20150317_005662	Matthew asked you to defeat the Pantos that might destroy the cable car again. Soil Poata's habitat in Margas Hills and lure them to defeat them. 
-QUEST_LV_0100_20150317_005663	Defeated Poatas as Matthew requested. Tell the good news to Matthew. 
+QUEST_LV_0100_20150317_005655	Lured the baby pantos with sugar beets. Listen to Molly's next plans. 
+QUEST_LV_0100_20150317_005656	$Talk to Guard Matthew
+QUEST_LV_0100_20150317_005657	$Retrieve Cable Car Parts
+QUEST_LV_0100_20150317_005658	$Guard Matthew asked for your help to find Cable Car Parts that the Zignuts swallowed around the Nepavy Grassland.
+QUEST_LV_0100_20150317_005659	$Found all the Cable Car Parts as Matthew asked. Give them to Matthew. 
+QUEST_LV_0100_20150317_005660	$Retrieve parts swallowed by Zignuts %s 
+QUEST_LV_0100_20150317_005661	$Defeat Poatas that threaten the cable car
+QUEST_LV_0100_20150317_005662	Matthew asked you to defeat the Pantos that might harm the cable car again. Soil the Poatas' habitat in Margas Hills and lure them to defeat them.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: Pantos or Poatas?

**Comment by Sunaries** [comment]
Looks good. Though it has merge conflicts.

**Comment by ttgmichael** [comment]
Merge conflict fixed :+1:

File: ETC.tsv [hunk 15]
```diff
@@ -12475,15 +12475,15 @@ ETC_20150714_012476	There are already jewels of prominence nearby
...
 ETC_20150714_012481	{memo X}You can clearly hear the sounds of devices!{nl}When you go to 5th area, a lubricant should be present there
-ETC_20150714_012482	You've found the lubricant!{nl}Go back to repair the demonic power supplying device
+ETC_20150714_012482	$You've found the lubricant!{nl}Go back to repair the Magic Supply Device
 ETC_20150714_012483	Now all pipes are working properly!{nl}Activate circulating purification device again
 ETC_20150714_012484	You've found all materials that are needed for the repair!{nl}Go back to the main purification device and repair
 ETC_20150714_012485	It seems that the demonic power is slowly leaking from the jewels of prominence!{nl}You should find the source and eliminate it fast
 ETC_20150714_012486	Stay in front of the valve to observe
 ETC_20150714_012487	Defeat Beakaras and progress 
-ETC_20150714_012488	You've defeated the Petrified Whale that is disturbing your search for the parts.{nl}Search for the other parts of the main device
+ETC_20150714_012488	$You've defeated the Stone Whale that interfered your search for parts.{nl}Search for extra parts for the Main Purifier
```
Comment (terrapin47) [inline ETC.tsv]: "interfered with"

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**